taxcalc/behavior.py

```diff
@@ -113,105 +94,144 @@ def has_response(self):
     def response(calc_x, calc_y):
         """
         Modify calc_y records to account for behavioral responses that arise
-        from the policy reform that involves moving from calc_x.policy to
-        calc_y.policy.
+          from the policy reform that involves moving from calc_x.policy to
+          calc_y.policy.
         Returns new Calculator object --- a deepcopy of calc_y --- that
-        incorporates behavioral responses to the reform.
+          incorporates behavioral responses to the reform.
+        Note: the use here of a dollar-change income elasticity (rather than
+          a proportional-change elasticity) is consistent with Feldstein and
+          Feenberg, "The Taxation of Two Earner Families" NBER Working Paper
+          No. 5155 (June 1995).  A proportional-change elasticity has become
```

Comment:
@MattHJensen said:
> Would the following be a more accurate statement than "has become more common since..."?
>
>> Gruber and Saez used a proportional change elasticity in "The elasticity of taxable income: evidence and implications", Journal of Public Economics 84:1-32 (2002) [see equation 2 on page 10].

It has been several weeks since I looked at [Emmanuel Saez, Joel Slemrod, and Seth H. Giertz, "The Elasticity of Taxable Income with Respect to Marginal Tax Rates: A Critical Review" _Journal of Economic Literature_ 2012](http://eml.berkeley.edu/~saez/saez-slemrod-giertzJEL12.pdf).  The way I remembered it was that many (most?) studies since 2002 have used the Gruber-Saez income effect specification.  But now that I look back at the 2012 JEL review, I can't seem to find a definitive statement to that effect.  Given that, I'll change the comment to read as you suggest.